conda/core/solve.py

```diff
@@ -1100,8 +1101,8 @@ def _setup_state(self):
         # Maybe conda already handles that beforehand
         # https://github.com/mamba-org/mamba/blob/fe4ecc5061a49c5b400fa7e7390b679e983e8456/mamba/mamba.py#L426-L485
 
-        prefix_data = PrefixData(self.prefix)
-        prefix_data.load()
+        mamba_prefix_data = MambaPrefixData(self.prefix)
```

Comment:
Could you please leave a comment here pointing to the C++ implementation of this class?

Reply:
Done. Interestingly, it's not that complicated. It's essentially a JSON loader.